cocotb/__init__.py

```diff
@@ -240,6 +240,23 @@ def _initialise_testbench(argv_):
 
     dut = cocotb.handle.SimHandle(handle)
 
+    try:
+        import pytest
+    except ImportError:
+        log.warning("Pytest not found, assertion rewriting will not occur")
+    else:
+        try:
+            # Install the assertion rewriting hook, which must be done before we
+            # import the test modules.
+            from _pytest.config import Config
+            from _pytest.assertion import install_importhook
+            pytest_conf = Config.fromdictargs([], {})
+            install_importhook(pytest_conf)
+        except Exception:
+            log.exception(
+                "Configuring the assertion rewrite hook using pytest {} failed. "
+                "Please file a bug report!".format(pytest.__version__))
+
```

Comment:
This isn't being covered and with these new warnings codecov is posting, it's getting really annoying. My best guess is that it's because we are starting coverage in this function. Should we move the coverage start code into a stub (that is `pragma: no cover`ed out) before loading the rest of the initialization?

Reply:
Well, we can always just ignore those warnings - it's not like we edit the initialization code that often. I wouldn't worry about it too much.